### DISCUSSION TITLE
Blocco Home Assistant

### DISCUSSION THREAD

**Discussion by Ale1975-Iron** [indomus/forum#302] [HUB: Home Assistant]
Ciao a tutti, è da qualche giorno che una volta al dì, si bloccca HA e sono costretto a riavviare, spegnendo brutalmente il raspberry.
Inizialmente è capitato di notte e credevo che fosse un problema dello snapshot notturno con l’add-on di Google Drive.
Oggi però è capitato nel pomeriggio.
Una volta riavviato, tutto funziona senza problemi.
Dove posso trovare delle info relative al blocco?
Può essere la SD che si sta danneggiando?
Grazie per il vostro sicuro aiuto

**Comment by snakuzzo** [comment]
Guarda il syslog

**Reply by Ale1975-Iron** [reply]
Ciao Skanuzzo, intendi il Log Viewer e il Log registri?
Avevo moltissimi errori relativi all’integrazione Vacuum... potrebbe essere questo il problema?
Grazie per l’aiuto

**Reply by snakuzzo** [reply]
Io parlavo del syslog su Linux `/var/log/syslog`
Se hai HassOS non so dirti...mai provato

**Reply by Ale1975-Iron** [reply]
Ciao Skanuzzo, io ho hassOs.
Proverò a disinstallare l’integrazione vacuum e vedo se cambia qualcosa.
Comunquemente grazie
